Possible improvements

linuspahl opened the pull request:
Hey Alex, ich habe mal die Readme's um meine learnings erweitert.

Da ich filebeat so unter OSX nicht nuten konnte, habe ich die Konfiguration um die Möglichkeit ergänzt jede beliebige Log Datei an Graylog zu senden. Mit ein paar Beispielen, wie man random logs generiert.

linuspahl committed:
Add readme for git-activities

linuspahl committed:
Add readme and enable filebeat setup to provide local log files

alex-konn reviewed (changes requested):
Danke, Linus! Das sieht doch alles sehr cool aus. Ich hätte einen einzigen Änderungswunsch: Bitte trenn' doch die random-logs-Geschichte von docker logs. Ich finde das Feature an sich cool, würde das aber separat halten und mit zwei getrennten filebeats-containern anbfrühstücken. das heißt, es gäbe docker-logs/ und random-logs/ statt docker-logs/random-logs. 
Das macht es erstens klarer, dass das zwei verschiedene Sachen sind und zweitens kannste dann auch verschiedene Inputs dafür nehmen und die Messages leichter unterschiedlich anreichern.
Ergibt das SInn für dich?

linuspahl commented:
Das hört sich gut an, das mache ich so!

linuspahl committed:
Create separate directory and filebeat container for random logs

alex-konn commented on random-logs/filebeat.yml:
```diff
@@ -0,0 +1,28 @@
+  hosts: 
+    - "127.0.0.1:5045"
+
+# Write Filebeat own logs only to file to avoid catching them with itself in docker log files
+logging.level: error
+logging.to_files: false
+logging.to_syslog: false
+loggins.metrice.enabled: false
+logging.files:
+  path: /var/log/filebeat
+  name: filebeat
+  keepfiles: 7
+  permissions: 0644
+ssl.verification_mode: none
```
Does this not conflict with the logging settings in the other filebeats container? I.e. wouldn't they overwrite each others' logs?

alex-konn committed:
Dynamically use the script path of run.sh

...as input path for random logs container

alex-konn committed:
Add link to flog to avoid confusion

...because `apt install flog` will install this:
https://www.unix.com/man-page/debian/1/flog/

alex-konn committed:
Added example for running flog docker container
